css/selectors/part.json

```diff
@@ -0,0 +1,47 @@
+{
+  "css": {
+    "selectors": {
+      "part": {
+        "__compat": {
+          "description": "<code>::part</code>",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/::part",
+          "support": {
+            "chrome": {
+              "version_added": null
+            },
+            "edge": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": "69",
+              "flags": [
+                {
+                  "name": "layout.css.shadow-parts.enabled",
+                  "type": "preference",
+                  "value_to_set": "true"
+                }
+              ]
+            },
+            "firefox_android": {
+              "version_added": false
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": null
+            },
+            "safari": {
+              "version_added": null
```

Comment:
What's the difference between version_added: null and version_added: false? Safari doesn't implement this.